File: Daily Pratice Question/C LANGUAGE/Question(19).c
// 19. Check if the array arr = [1, 2, 3, 4, 5] is
//  sorted in ascending order, and print true or false.

#include<stdio.h>
  int main(){

          int arr[] = {1,2,3,4,5};
      int i;
       int ascending;
    int len = sizeof(arr)/sizeof(arr[0]);
    for(i=0;i<len-1;i++){
        if(arr[i]<=arr[i+1]){
           ascending++;
        }
            
        }if(ascending==len-1){
            printf("true array is in ascending order");
            
        }else{
             printf("false array is not in ascending order");
        }
    
return 0;
 }

File: Daily Pratice Question/JAVASCRIPT/question(19).js
// 19. Check if the array arr = [1, 2, 3, 4, 5] is 
// sorted in ascending order, and print true or false.


let arr = [1, 2, 3, 4, 5]
let flag=0
for(i=0;i<arr.length-1;i++){
    if(arr[i]<arr[i+1]){
        flag++
    }
}if(flag==arr.length-1){
    console.log("true given array is asorted asecending order")
}else{
     console.log("false given array is not  sorted in asecending order")
}